homeassistant/components/alarm_control_panel/translations/af.json

```diff
@@ -1,17 +1,18 @@
 {
     "state": {
-        "_": {
-            "armed": "Gewapen",
-            "armed_away": "Gewapend weg",
-            "armed_custom_bypass": "Gewapende pasgemaakte omseil",
-            "armed_home": "Gewapend tuis",
-            "armed_night": "Gewapend nag",
-            "arming": "Bewapen Tans",
-            "disarmed": "Ontwapen",
-            "disarming": "Ontwapen Tans",
-            "pending": "Hangende",
-            "triggered": "Geaktiveer"
-        }
+      "_": {
```

Comment:
Please don't change translation files.

Reply:
Please explain; these files are used by the alarm_control_panel component. Do you have a translation team that takes care of this?

Reply:
The translation files are synced automatically from Lokalise. The source of truth is `strings.json`.

You can read more here:
https://developers.home-assistant.io/docs/translations